app/controllers/works_controller.rb

```diff
@@ -196,7 +200,8 @@ def editable?
   end
 
   def set_presenter
-    @work_presenter = WorkPresenter.new(work: @work, work_form: @work_form, version_status: @version_status)
+    @work_presenter = WorkPresenter.new(work: @work, work_form: @work_form, version_status: @version_status,
+                                        user_version: Sdr::Repository.latest_user_version(druid:))
```

Comment:
set the `user_version` for display in the presenter, value comes from DSA client call which occurs in the Sdr::Repository object